src/components/Text/Text.jsx

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import cn from 'classnames';
+// @flow
+/* eslint react/prop-types: 0 */
+import * as React from 'react';
+import cx from 'classnames';
 import style from './style.scss';
-import { getDataAttrs, getDataProps } from '../../dataUtils';
+import type { Data } from '../../types';
+import { getDataAttrs } from '../../dataUtils';
 
-class Text extends React.Component {
+export type Props = {
+  children: React.Node,
+  data?: Data,
+  large?: boolean,
+  style?: Object,
+  type?: string,
+};
+
+const types = {
+  PRIMARY: 'primary',
+  SECONDARY: 'secondary',
+};
+
+class Text extends React.Component<Props> {
+  static defaultProps = {
+    style: { width: 200, height: 'auto' },
```

Comment:
If you move this to `.scss` file, you could remove `defaultProps` and make this a functional component.

Reply:
we want this to be in `defaultProps` so that it can be overwritten if `style` is passed in. i don't think we should remove this without refactoring ssv2 code to pass in width rather than assuming a default width.